### PR TITLE
registerSchema()

### LINKED ISSUE #2983: connectors & shopify connectors I18n
Related to https://github.com/reactioncommerce/reaction/issues/2641

Missing i18n translations for many parts of the `connectors` and `connectors-shopify` packages.

Missing translations on:
- Tooltip on icon toolbar
- Titles on panels
- Form fields related to API key/secret
- Translation file copied from shipping in `connectors` package, needs to be updated with proper namespace and all duplicate keys removed.

### PR DESCRIPTION
**Pull request by kieckhafer**
Resolves #2983 
Documention here: https://github.com/reactioncommerce/reaction-docs/pull/296

- Create new `registerSchema()` to register schemas in an object
- Use registered schemas to apply i18n translations to labels of forms created with autoform && SimpleSchema

### CODE REVIEW

**Comment by spencern** [comment]
@aaronjudd yes please.

File: imports/plugins/core/collections/lib/registerSchema.js [hunk 1]
```diff
@@ -0,0 +1,25 @@
...
+ *
+ * @returns {Empty} no return. Adds schema to Schemas object
+ */
+export function registerSchema(name, schema) {
+  if (!name || !schema) {
+    throw new Error("A name and schema object are required for registerSchema");
+  }
+
+  // store the component in the table
+  Schemas[name] = schema;
```
Comment (mikemurray) [inline imports/plugins/core/collections/lib/registerSchema.js]: Should we match the style of `registerComponent` and do `Schemas[name] = { name, schema }` ?

**Comment by kieckhafer** [comment]
@aaronjudd fixed.

**Comment by spencern** [comment]
@aaronjudd merging because your approval + tests pass.

**Pull request merged**